src/extractor/guidance/intersection_handler.cpp

```diff
@@ -375,19 +391,29 @@ std::size_t IntersectionHandler::findObviousTurn(const EdgeID via_edge,
 
```

Comment:
This function has grown in size dramatically, do you think there is a way to split the different cases out?